AmplifyPlugins/Auth/Sources/AWSCognitoAuthPlugin/StateMachine/CodeGen/States/DebugInfo/CredentialStoreState+Debug.swift

```diff
@@ -7,31 +7,37 @@
 
 import Foundation
 
-extension CredentialStoreState {
+extension CredentialStoreState: CustomDebugStringConvertible {
     var debugDictionary: [String: Any] {
-        let stateTypeDictionary: [String: Any] = ["CredentialStoreState": type]
         var additionalMetadataDictionary: [String: Any] = [:]
```

Comment:
_Outside of the lines you've touched in this PR, so this is a non-blocking recommendation._

Since we're exhaustively checking state in the switch, consider declaring this an uninitialized `let` to enforce assignment and prevent mutation.
```swift
let additionalMetadataDictionary: [String: Any]
```